ocs_ci/ocs/pgsql.py

```diff
@@ -337,6 +338,31 @@ def respin_pgsql_app_pod(self):
             resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
         )
 
+    def get_pgbech_pod_status_table(self, pgbench_pods):
+        """
+        Get pgbench pod data and print results on a table
+
+        Args:
+            pgbench pods (list): List of pgbench pods
+
+        """
+        pgbench_pod_table = PrettyTable()
+        pgbench_pod_table.field_names = ['pod_name', 'scaling_factor', 'num_clients', 'num_threads',
```

Comment:
break the line according to 
https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md